<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connecter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>391</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Il faut que vous etes un employe de notre entreprise pour que vous pouvez connecter Ã  notre programme !&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(143, 143, 143);
border:2px solid #000;
}
QPushButton {
     color: #fff !important;
text-transform: uppercase;
text-decoration: none;
background: #e12d4e;
padding: 20px;
border-radius: 5px;
display: inline-block;
border: none;
transition: all 0.4s ease 0s;
 }
 QPushButton:hover {
     background: #434343;
letter-spacing: 1px;
-webkit-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);
-moz-box-shadow: 0px 5px 40px -10px rgba(0,0,0,0.57);
box-shadow: 5px 40px -10px rgba(0,0,0,0.57);
transition: all 0.4s ease 0s;
 }</string>
    </property>
    <property name="text">
     <string>connecter</string>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Identifiant</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WELCOME TO TECHNOR5IS</string>
    </property>
   </widget>
   <widget class="QLabel" name="mdp">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mdp_edit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(143, 143, 143);
border:2px solid #000;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="session">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>435</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Garder ma session ouverte</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_edit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
background-color: rgb(143, 143, 143);
border:2px solid #000;
}</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="etat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_mdp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>311</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Afficher mot de passe.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
padding: 1px;
border-style: solid;
border: 1px solid gray;
border-radius: 12px;
background-color: rgb(143, 143, 143);
border:2px solid #000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>350</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Responsable produit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Responsable rayon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Responsable employe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Responsable parking</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Responsable clients</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>350</y>
      <width>146</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Domaine de travail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="background2_2">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>-10</y>
      <width>1241</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background2_2</zorder>
   <zorder>connecter</zorder>
   <zorder>id</zorder>
   <zorder>name</zorder>
   <zorder>mdp</zorder>
   <zorder>mdp_edit</zorder>
   <zorder>session</zorder>
   <zorder>id_edit</zorder>
   <zorder>progressBar</zorder>
   <zorder>etat</zorder>
   <zorder>check_mdp</zorder>
   <zorder>comboBox</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
